tensorboard/plugins/text/text_plugin.py

```diff
@@ -234,9 +235,15 @@ def is_active(self):
       if any(self._index_cached.values()):
         return True
 
+    if bool(self._multiplexer.PluginRunToTagToContent(metadata.PLUGIN_NAME)):
```

Comment:
I think having `if bool(expr)` is redundant - you can just do `if expr`.

Reply:
Done.